examples/hnas/example_mlp.py

```diff
@@ -0,0 +1,261 @@
+"""
+In this is example we show how we can optimize the hyperparameters of a simple 2 layer
+feed forward neural networks on MNIST. We optimize the following hyperparameter:
+
+- number of units first layer
+- number of units second layer
+- dropout first layer
+- dropout second layer
+- learning rate of ADAM
+- batch size
+- weight decay
+
```

Comment:
Consider linking this pytorch tutorial as well: https://auto.gluon.ai/stable/tutorials/torch/hpo.html

and briefly describing what the differences are. 

Why are both tutorials useful? It appears to me a key difference is the emphasis on hyperband-style HPO in this example, in which case the comment at the top should probably make this clear.

Reply:
you're right, the main focus here is successive-halving based HPO. I made this more clear

Reply:
We could link this example in the Pytorch tutorial